_data-prepper/pipelines/configuration/processors/trace-peer-forwarder.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: trace peer forwarder
```

Comment:
@natebower @carolxob Shouldn't we treat these headings like all other headings sentence case with first word initial capitalized? Otherwise, should the headings read something like "The `trace peer forwarder` processor?"